.github/workflows/rc.yml

```diff
@@ -105,37 +357,40 @@ jobs:
           else
             rc=$(date +%Y%m%d)
           fi
-          VERIFY_DEFAULT=0 \
-            VERIFY_SOURCE=1 \
-            dev/release/verify_rc.sh "${version}" "${rc}"
+          tar -xf ${tar_gz}
+          export VERIFY_DEFAULT=0
+          export VERIFY_BINARY=1
+          export VERIFY_SOURCE=1
+          cd apache-arrow-java-*
+          mv ../${tar_gz} ./
+          mv ../release-binaries binaries
+          dev/release/verify_rc.sh "${version}" "${rc}"
   upload:
     name: Upload
-    if: github.ref_type == 'tag'
+    # if: github.ref_type == 'tag'
```

Comment:
I'll revert this after I confirmed that this job works.

Reply:
Reverted. This worked: https://github.com/apache/arrow-java/actions/runs/12781206977/job/35629480932#step:3:17